clickplc/driver.py

```diff
@@ -47,11 +79,21 @@ async def get(self, address: str) -> dict:
         This uses the ClickPLC's internal variable notation, which can be
         found in the Address Picker of the ClickPLC software.
         """
+        if address is None:
+            if not self.tags:
+                raise ValueError('An address must be supplied to get if tags were not '
+                                 'provided when driver initialized')
+            results = {}
+            for category, address in self.active_addresses.items():
+                results.update(await getattr(self, '_get_' + category)(address['min'], address['max']))
+            return {nickname: results[tag_info['id'].lower()]
```

Comment:
Could we call `tag` or `tagname`? No need to have ambiguous terminology.

Reply:
So the click tags files calls these things nicknames. We can enforce our own consistency I suppose

Reply:
That's weird. Does Click always call it nicknames and we're adding the "tags" naming from our use with Productivity PLC?

If so, we should call them nicknames everywhere and remove all reference to tags.